File: doc/src/scripts/client/python/DeferredCall.py
import krpc
from krpc import defer

conn = krpc.connect()
vessel = conn.space_center.active_vessel
sc = conn.space_center


# warp_to pauses execution and resumes on a later tick, so the function
# starts it with defer and carries on without waiting for it
def warp_and_arm():
    defer(sc.warp_to(sc.ut + 60))
    vessel.control.rcs = True


conn.run_function(warp_and_arm)

File: doc/src/scripts/client/csharp/DeferredCall.cs
using System;
using KRPC.Client;
using KRPC.Client.Services.SpaceCenter;

class Program {
    public static void Main () {
        var connection = new Connection ();
        var spaceCenter = connection.SpaceCenter ();

        // WarpTo pauses execution and resumes on a later tick, so the function
        // starts it with Function.Defer and returns without waiting for it
        connection.RunFunction (
            () => Function.Defer (
                () => spaceCenter.WarpTo (spaceCenter.UT + 60, 100000, 2)));

        Console.WriteLine ("The warp is running, and the call has returned");
    }
}
